Makefile

```diff
@@ -0,0 +1,23 @@
+export GO111MODULE = on
+
+build_tags := $(strip $(BUILD_TAGS))
+BUILD_FLAGS := -tags "$(build_tags)"
+
+OUT_DIR = ./build
+
+MODULE=github.com/medibloc/panacea-dep-consumer
```

Comment:
If you don't need this var, delete it.

Reply:
You probably don't need this. This variable was used for proto.